calm/draft/2024-10/meta/flow.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://calm.finos.org/draft/2024-10/meta/flow.json",
+  "title": "Business Flow Model",
+  "description": "Defines business flows that relate to technical architectures, allowing mapping of flows to technical components and attaching control requirements.",
+  "type": "object",
+  "properties": {
+    "unique-id": {
+      "type": "string",
+      "description": "Unique identifier for the flow"
+    },
+    "name": {
+      "type": "string",
+      "description": "Descriptive name for the business flow"
+    },
+    "description": {
```

Comment:
Worth calling it "purpose" or "intent"?

Reply:
I think description is consistent with the rest of the schema.